### PR TITLE
Completed Day6 Problems: Please review

### PR DESCRIPTION
**Pull request by ShaivyT**

### CODE REVIEW

File: Problem3.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// Time Complexity : O(n)
+// Space Complexity : O(n)
```
Comment (samarthjj) [inline Problem3.java:2]: O(52) or O(1), the input will only have Upper and lower case letters.

**Comment by samarthjj** [comment]
Good work Shaivy, Please review the inline comments.
Please complete the **Contiguous Array** Problem as well.

**Comment by samarthjj** [comment]
Good work Shaivy, Please review the inline comments.
Please complete the **Contiguous Array** Problem as well.

**Comment by ShaivyT** [comment]
Please check now. Sorry I forgot to push Problem 2.

File: Problem3.java (hunk 1 above)
Comment (ShaivyT) [reply Problem3.java:2]: Can you please explain? I mean is it constant?

**Pull request closed without merging**